yarn.lock

```diff
@@ -15872,6 +15880,16 @@ fsevents@^1.2.7:
   languageName: node
   linkType: hard
 
+"micromatch@npm:^4.0.4":
```

Comment:
can you dedupe this?

`yarn dedupe` almost works, but if you use it you also need to roll back `source-map-support` due to #9147